cmsdist-comp-pr-process.py

```diff
@@ -1,34 +1,38 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 from __future__ import print_function
+import sys
 from sys import exit,argv
 from re import match
-from github import Github
 from os.path import expanduser, dirname, abspath, join, exists
-from optparse import OptionParser
+from argparse import ArgumentParser
 from socket import setdefaulttimeout
+
+from github_utils import get_pr, get_issue_comments, merge_pr, edit_issue, \
+  create_issue_comment
+
 setdefaulttimeout(120)
 SCRIPT_DIR = dirname(abspath(argv[0]))
 
-def process_pr(gh, repo, issue, dryRun):
+def process_pr(repo: str, issue: dict, dryRun: bool):
```

Comment:
@iarspider , this does not look right, you are passing `prId` as 2nd arg while signature here expacs a dict.